data/queenstack.json)

```diff
@@ -0,0 +1,22 @@
+
+ { "name": "Agada Queen",
+  "type": "personal",
+  "bio": "Data scientist **Business Analyst**. Technical writer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/maquinbeaury",
+       "icon": "FaTwitter"
```

Comment:
```suggestion
       "icon": "FaTwitter"
       "color": "#00ACEE"
```

Reply:
Hey @CBID2, we don't need the color attribute for the popular links like Twitter, Linkedin, Youtube and many more. Colors for these links is already configured in [`icons.json`](https://github.com/EddieHubCommunity/LinkFree/blob/main/config/icons.json)